examples/kitty/client/kitty.py

```diff
@@ -18,7 +18,7 @@
 reader_stream = None
 writer_stream = None
 
-IP_ADDRESS = "172.16.0.2"
+IP_ADDRESS = "10.13.1.17"
```

Comment:
Not sure if this change should be included

Reply:
Thanks for catching that!